### PR TITLE
Docs: Update timeseries-dimensions.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor edit

### CODE REVIEW

**Review by kylebrandt — APPROVED** [review]

File: docs/sources/getting-started/timeseries-dimensions.md [hunk 1]
```diff
@@ -8,9 +8,9 @@ weight = 600
 
 # Time series dimensions
 
-At the end of the ["Time series databases" section of "Introduction to time series"]({{< relref "timeseries.md#time-series-databases" >}}), the concept of _labels_, also called _tags_, is introduced:
+In [Introduction to time series]({{< relref "timeseries.md#time-series-databases" >}}), the concept of _labels_, also called _tags_, is introduced:
```
Comment (achatterjee-grafana) [inline docs/sources/getting-started/timeseries-dimensions.md:11]: Did not find any reference to `_labels_` or `_tags_` in the topic: Introduction to time series. FYI.

**Review by achatterjee-grafana — COMMENTED** [review]
Added a comment.

File: docs/sources/getting-started/timeseries-dimensions.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/getting-started/timeseries-dimensions.md:11]: @kylebrandt, are you sure this sentence is correct? Should we remove it?

Comment (kylebrandt) [reply docs/sources/getting-started/timeseries-dimensions.md:11]: It is there:

image

image

that ... table? doesn't look so well :-P

Comment (achatterjee-grafana) [reply docs/sources/getting-started/timeseries-dimensions.md:11]: Ok. I searched using "_labels_" and "_tags_", that is why there was no result for me.

**Pull request merged**